src/azul/deployment.py

```diff
@@ -480,10 +480,17 @@ def _log_client_response(self,
                              **kwargs: Any
                              ) -> Optional[AWSResponse]:
         event_name = self._shorten_event_name(event_name, self._response_event_name)
-        response = kwargs['response_dict']
-        boto3_log.info('%s:\tGot %s response', event_name, response['status_code'])
-        message = http_body_log_message(boto3_log, 'response', response.get('body'))
-        boto3_log.info('%s:\t%s', event_name, message)
+        response, exception = kwargs.get('response_dict'), kwargs.get('exception')
+        if exception is None:
+            if response is None:
+                boto3_log.info('%s:\tGot no response', event_name)
+            else:
+                boto3_log.info('%s:\tGot %s response', event_name, response['status_code'])
+                message = http_body_log_message(boto3_log, 'response', response.get('body'))
```

Comment:
Did we observe both cases, `body` absent and `body` set to `None`?

Reply:
Discussed during PL